### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by nebnes** [security-union/av1-streaming#13]
Hello, 

I have an error when I try to build : 

```
error: failed to run custom build command for `v4l2-sys-mit v0.2.0`

Caused by:
  process didn't exit successfully: `[...]/av1-streaming/target/release/build/v4l2-sys-mit-9df7d216a778e481/build-script-build` (exit status: 101)
  --- stderr
  /usr/include/linux/videodev2.h:60:10: fatal error: 'sys/time.h' file not found
  /usr/include/linux/videodev2.h:60:10: fatal error: 'sys/time.h' file not found, err: true
  thread 'main' panicked at 'Failed to generate bindings: ()', [...]/.cargo/registry/src/github.com-1ecc6299db9ec823/v4l2-sys-mit-0.2.0/build.rs:10:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

**Comment by darioalessandro** [comment]
Hi @nebnes !! 

Happy to help 😄 can you please include:

1. Build command.
2. OS
3. Docker version

This will help me to speed up the triage.

**Comment by nebnes** [comment]
Hi @darioalessandro, 

Thanks !

Here is my setup : 

1. Build command : 
`sh build-raspy.sh`
2. OS : 
Ubuntu 20.04.5 LTS
3. Docker version : 
Docker version 20.10.18

**Comment by darioalessandro** [comment]
awesome, let me test

**Comment by darioalessandro** [comment]
Based on the os version, I am almost sure that the versions of `libclang-dev` and `libv4l-dev` that `v4l2-sys-mit` expects is different.

I was able to reproduce it in my linux box with ubuntu 20.

<img width="3120" alt="Screen Shot 2022-10-18 at 5 35 48 PM" src="https://user-images.githubusercontent.com/1176339/196549510-5dd7fb0b-7fe4-4d42-bf51-47bc91b15e2c.png">

I have another  box with ubuntu 22 and everything worked.

```
darioalessandro@pop-os:~/Documents/2022/av1-streaming$ v4l2-ctl --version
v4l2-ctl 1.22.1
```

We have 2 options:

1. Upgrade Ubuntu to at least Ubuntu 21 and upgrade the `libv4l-dev` version.
2. Find a way to install libv4l-dev version 1.22.1 on ubuntu 20, I looked around and did not find a way.
